src/main/java/de/zalando/aruha/nakadi/exceptions/NoStreamingSlotsAvailable.java

```diff
@@ -0,0 +1,14 @@
+package de.zalando.aruha.nakadi.exceptions;
+
+import javax.ws.rs.core.Response;
+
+public class NoStreamingSlotsAvailable extends NakadiException {
+    public NoStreamingSlotsAvailable(final int totalSlots) {
+        super("No free slots for streaming available. Total slots: " + totalSlots);
+    }
+
+    @Override
+    protected Response.StatusType getStatus() {
+        return Response.Status.BAD_REQUEST;
```

Comment:
There are 3 candidates for that:
- 429 TOO MANY REQUESTS (which is not the part of official rfc2616)
- 409 Conflict (part of rfc, but too broad)
- 418 I'm a teapot (I didn't used this code anywhere, and want to try, and it's not a part of rfc)

429 Makes more sense, but I believe that it's not supported in different clients (like python one). So probably the best way is to use 409.